src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -169,52 +154,52 @@ struct Hydrogen
  *
  * Usage: Add a flag to the list of particle flags that has the following structure
  *
- *        ionizer<IonizationModel<Species2BCreated> >
+ *        ionizer< IonizationModel< Species2BCreated > >
  */
 
-typedef bmpl::vector<
-    particlePusher<UsedParticlePusher>,
-    shape<UsedParticleShape>,
-    interpolation<UsedField2Particle>,
-    current<UsedParticleCurrentSolver>,
-    massRatio<MassRatioIons>,
-    chargeRatio<ChargeRatioIons>
-> ParticleFlagsIons;
+using ParticleFlagsIons = bmpl::vector<
+    particlePusher< UsedParticlePusher >,
+    shape< UsedParticleShape >,
+    interpolation< UsedField2Particle >,
+    current< UsedParticleCurrentSolver >,
+    massRatio< MassRatioIons >,
+    chargeRatio< ChargeRatioIons >
+>;
 
 /*define specie ions*/
-typedef Particles<
+using PIC_Ions = Particles<
     ParticleDescription<
-        bmpl::string<'i'>,
+        bmpl::string< 'i' >,
         SuperCellSize,
-        DefaultAttributesSeq,
```

Comment:
Right the reason is that the pusher extention can not differ between species with and without `momentumPrev1`.
To change this wee need to cleanup the pusher extension first.